docs/sources/setup-grafana/configure-docker.md

```diff
@@ -57,31 +89,152 @@ The following settings are hard-coded when launching the Grafana Docker containe
 
-It's possible to supply Grafana with configuration through files. This works well with [Docker Secrets](https://docs.docker.com/engine/swarm/secrets/) as the secrets by default gets mapped into `/run/secrets/<name of secret>` of the container.
+Example:
+
+The following example shows how to build a customized Grafana Docker image that includes the Image Renderer plugin.
+
+```bash
+#go to the folder
+cd packaging/docker/custom
 
-You can do this with any of the configuration options in conf/grafana.ini by setting `GF_<SectionName>_<KeyName>__FILE` to the path of the file holding the secret.
+#running the build command
+docker build \
+--build-arg "GRAFANA_VERSION=latest" \
+--build-arg "GF_INSTALL_IMAGE_RENDERER_PLUGIN=true" \
+-t grafana-custom .
 
-For example, you could set the admin password this way:
+# running the docker run command
+docker run -d -p 3000:3000 --name=grafana grafana-custom
+```
+
+## Logging
+
+By default, Docker container logs are directed to `STDOUT`, which is a common practice in the Docker community. You can change this by setting a different [log mode]({{< relref "../../../configure-grafana/#mode" >}}) such as `console`, `file`, or `syslog`. You can choose to use one or more modes by separating them with spaces, for example, `console file`. By default, both `console` and `file` modes are enabled.
+
+Example:
+
+The following example runs Grafana using the `console file` log mode that is set in the `GF_LOG_MODE` environment variable
+
+```bash
+# Run Grafana while logging to both standard out 
+# and /var/log/grafana/grafana.log
+
+docker run -p 3000:3000 -e "GF_LOG_MODE=console file" grafana/grafana-enterprise
+```
+# Configure Grafana with Docker Secrets
+
+Using configuration files, you can provide sensitive information such as secrets and login credentials to Grafana. This method works particularly well with Docker Secrets, as the secrets are automatically mapped to the `/run/secrets/` location within the container. 
+
+You can apply this technique to any configuration options in `conf/grafana.ini` by setting `GF_<SectionName>_<KeyName>__FILE` to the file path that contains the secret information.
+
+The following example demonstrates how to set the admin password:
 
 - Admin password secret: `/run/secrets/admin_password`
 - Environment variable: `GF_SECURITY_ADMIN_PASSWORD__FILE=/run/secrets/admin_password`
 
-## Configure AWS credentials for CloudWatch Support
+## Configure Docker secrets credentials for AWS CloudWatch
+
+Grafana ships with built-in support for [Amazon CloudWatch datasource](https://grafana.com/docs/grafana/latest/datasources/aws-cloudwatch/), where need to provide information such as AWS ID-Key, secret access key, region etc. For which you can use the Docker secrets to achieve it.
```

Comment:
```suggestion
Grafana ships with built-in support for the [Amazon CloudWatch datasource]({{< relref "../datasources/aws-cloudwatch/" >}}). To configure the data source, you must provide information such as the AWS ID-Key, secret access key, region, and so on. You can use Docker secrets as a way to provide this information.
```

Reply:
I added a suggestion, but don't really understand it ;) - We are telling the user that they must provide the information and that Docker secrets can also provide the information. Can you pls help clarify?